src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/SavePipelineToJSONAndLoad.java

```diff
@@ -0,0 +1,57 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.foundations.file.LocalFile;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.action.convert.ConvertStringToNumberCommand;
+import com.northconcepts.datapipeline.foundations.pipeline.action.transform.AddFieldsCommand;
+import com.northconcepts.datapipeline.foundations.pipeline.action.transform.RenameFieldsCommand;
+import com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.output.ExcelPipelineOutput;
+import com.northconcepts.datapipeline.job.Job;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class SavePipelineToJSONAndLoad {
```

Comment:
Where are we saving and restoring this pipeline from JSON?
Let's rename this class to `SaveAndRestorePipelineFromJson`.

Reply:
Just re-added the code.

Reply:
I don't see the rename, was this class renamed?

Reply:
Sorry my mistake. Pushed the rename